src/descriptors-device.adb

```diff
@@ -363,12 +363,14 @@ package body Descriptors.Device is
 
          if Slice (Device.Description, 1, 5) = "STM32" then
             --  ??? Workaround for the STM32F* svd files that do not define the
-            --  FPU Interrupt
-            Interrupts.Append
-              ((Name        => To_Unbounded_String ("FPU"),
-                Description => To_Unbounded_String
-                  ("FPU global interrupt"),
-                Value       => 81));
+            --  FPU Interrupt. The STM32F0 series do not have an FPU.
+            if not (Slice (Device.Description, 1, 7) = "STM32F0") then
```

Comment:
You should use 'and then not' here, to clearly show the condition. Also you should use /= instead of not =. Finally, the parenthesis are superfluous when using the inequality test. Finally, this should look like:

`
   if Slice (Device.Description, 1, 5) = "STM32" 
      and then Slice (Device.Description, 6, 7) /= "F0"
`